src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -245,6 +245,12 @@ class DefaultStorageKeyAllocator
     // when new fabric is created, this list needs to be updated,
     // when client init DefaultICDClientStorage, this table needs to be loaded.
     static StorageKeyName ICDFabricList() { return StorageKeyName::FromConst("g/icdfl"); }
+
+    // Energy EVSE cluster
+    // Need a key to store the Charging Preference Targets which is a TLV of list of lists
```

Comment:
Needs to be kept somewhere private since its not in sdk area.
Can just use const char.

Reply:
Also need delete all storage when last fabric is deleted as part of factory reset.
Make an API but not until 1.4

Reply:
I have fixed the comment above by moving the the EVSE targets key to be a private member of EnergyEvseTargetsStore